tox.ini

```diff
@@ -1,22 +1,25 @@
 [tox]
 envlist =
-    {py27,py35,py36}-django{18,19,110,111}
-    {py35,py36,py37}-django{20,21}
     py27-django17
+    {py27,py35,py36}-django{18,19,110,111}
+    {py35,py36,py37}-django{20,21,22}
+    py38-django{22,30}
 
 [testenv]
 commands = ./test.sh {posargs}
 deps =
-     django-extensions==1.6.1
-     django17: django==1.7.11
-     django18: django==1.8.18
-     django19: django==1.9.13
-     django110: django==1.10.7
-     django111: django==1.11.2
-     django20: django==2.0.9
-     django21: django==2.1.3
-     six==1.10.0
-     sqlalchemy==1.0.12
+    django17: django~=1.7.0
+    django18: django~=1.8.0
+    django19: django~=1.9.0
+    django110: django~=1.10.0
+    django111: django~=1.11.0
+    django20: django~=2.0.0
+    django21: django~=2.1.0
+    django22: django~=2.2.0
+    django30: django~=3.0.0
+    six
+    sqlalchemy
+    django-extensions
```

Comment:
not sure whether that's related with the test failures, but you're unpinning six, sqla and django-extensions here...

Reply:
It's not related to the test failures, I just think it's silly to pin dependencies for tests in general.